More Barsigns, Service Holosign

KingOfThePing opened the pull request:
After a brief discourse in Discord #general channel it was decided to add some more barsigns and a new one (I had sitting around for 500 years on my PC). Three new barsigns were added:
![image](https://github.com/user-attachments/assets/d8a4f87d-5cf2-458a-b584-a397a985db2a)

With the following descriptions:
![image](https://github.com/user-attachments/assets/4c1d5893-2504-4174-8066-d593259a1e2c)


The new holosign looks like this:
![image](https://github.com/user-attachments/assets/2f3ecd5d-5dc7-4768-8c08-500a4388d6ea)

Has a nice little animation. Signs and buttons are mapped into bar and kitchen. Some really minor shuffling was done for the signs. Tested locally and it works on my machine (💯).

Please enjoy.
### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| barsigns.dmi| Unknown (BayStation12) | AGPL v3.0 |

KingOfThePing commented:
!review

SleepyGemmy commented on code/game/machinery/holosign.dm:
```diff
@@ -48,11 +48,23 @@
+	desc = "A small wall-mounted holographic projector. This one reads OPEN."
+
+/obj/machinery/holosign/service/update_icon()
+	if (!lit)
```
```suggestion
	if(!lit)
```

SleepyGemmy commented on code/game/objects/structures/barsign.dm:
```diff
@@ -145,6 +145,20 @@
+	name = "We Are Open!"
+	icon_state = "We Are Open!"
+	desc = "The bar's sign flashing 'We Are Open!' indicating that it is indeed open for business."
+	desc_extended = "Sometimes it doesn't need a fancy sign, slogan or flashly colours to draw in people to the bar. The service and booze usually does. This sign here simply let's crew know that they are getting both here, right now."
```
```suggestion
	desc_extended = "Sometimes it doesn't need a fancy sign, slogan, or flashy colours to draw in people to the bar. The service and booze usually does. This sign here simply lets crew know that they are getting both here, right now."
```

SleepyGemmy commented on code/game/objects/structures/barsign.dm:
```diff
@@ -145,6 +145,20 @@
+/singleton/sign/double/bar/free_drinks
+	name = "Free Drinks!"
+	icon_state = "Free Drinks!"
+	desc = "Either the XO or the Captain were in a really good mood, someone's pranking the crew or it is your lucky day. Cheers!"
```
```suggestion
	desc = "Either the XO or the Captain were in a really good mood, someone's pranking the crew, or it is your lucky day. Cheers!"
```

SleepyGemmy reviewed (changes requested):
misplaced light.

![image](https://github.com/user-attachments/assets/4760f3e8-8ff8-452c-a810-761ab0dec8cb)

not a fan of the camera being inside the shutter. if there's no other way, then it's fine.

![image](https://github.com/user-attachments/assets/8541f387-d37d-48a7-b508-edcb455e1d50)

KingOfThePing commented:
> misplaced light.
> 
> ![image](https://private-user-images.githubusercontent.com/99297919/463944897-4760f3e8-8ff8-452c-a810-761ab0dec8cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTIwNzE3MDksIm5iZiI6MTc1MjA3MTQwOSwicGF0aCI6Ii85OTI5NzkxOS80NjM5NDQ4OTctNDc2MGYzZTgtOGZmOC00NTJjLWE4MTAtNzYxYWIwZGVjOGNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzA5VDE0MzAwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwZTcyOWQ1ODZlYWUwZDgyMzg1OTc3ODM2ZWZlNzIxY2I4N2NiOTNhYjc5OGNmNDI0ZTI4ZTQ2MzU0YThmN2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SnIF-r0VNf6b9jFbTcHQzBtGVjGAEF6rdLlHuw55tWQ)
> 
> not a fan of the camera being inside the shutter. if there's no other way, then it's fine.
> 
> ![image](https://private-user-images.githubusercontent.com/99297919/463945233-8541f387-d37d-48a7-b508-edcb455e1d50.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTIwNzE3MDksIm5iZiI6MTc1MjA3MTQwOSwicGF0aCI6Ii85OTI5NzkxOS80NjM5NDUyMzMtODU0MWYzODctZDM3ZC00OGE3LWI1MDgtZWRjYjQ1NWUxZDUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzA5VDE0MzAwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzNDhmNjMyOGE5NDM4MjNmNzhiZTU2NTJmMjBhOTAwYjZiMjE0NzMwOTRmNmIxNWMwOGViZThmZDkyZDk3YjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8TOnymIDoEyHWePL34nrxLacdNXjD_dCmm__aRM-fUE)

I see the issue with the camera. I moved it one up because it would clip into the new sign otherwise (I was asked if I could place the sign there). To be fair I didnt test if you can still reach the camera or if the camera coverage now shows nothing. I am open for suggestions where to place it instead. 

Regarding the light I am not sure what you mean with misplaced? I just moved it up. It doesn't clip into the maintenance panel, if that's the issue.

KingOfThePing commented:
All changes adressed
